tests/test_meteodatalab/test_flexpart.py

```diff
@@ -89,6 +89,14 @@ def test_flexpart(data_dir, fieldextra):
         }
     )
 
+    assert ds_out["omega"].parameter == {
+        "centre": "ecmf",
```

Comment:
This matches the output of eccodes?

Reply:
Yes!! the "w" is missing in `ECMWF`, I don't know if it is on purpose. But this is retrieved with `metadata.as_namespace("parameter")` which is from `earthkit-data` so I don't think we can correct anything...